src/arguments.rs

```diff
@@ -64,9 +65,11 @@ App::new("Drogue Command Line Tool")
     .about("Allows to manage drogue apps and devices in a drogue-cloud instance")
     .arg(config_file_arg)
     .arg(url_arg)
+    .setting(AppSettings::SubcommandRequiredElseHelp)
```

Comment:
Why not adding : `setting(AppSettings::SubcommandRequiredElseHelp)` to the `App` object at the root level instead of adding it to every subcommand ?
(maybe it does not works recursively, but i think it does at it is not mentionned in the documentation)

Reply:
Tried this but it doesn't work recursively.